psycopg/psycopg/_transformer.py

```diff
@@ -11,6 +11,8 @@
 from . import abc
 from ._cmodule import _psycopg
 
+__all__ = ("Transformer",)
```

Comment:
Why is this necessary? It seems something unneeded to maintain.

Reply:
I received an error with mypy without it:

https://github.com/psycopg/psycopg/actions/runs/11784504225/job/32823815099#step:7:12

> Module "psycopg._transformer" does not explicitly export attribute "Transformer"  [attr-defined]

This may be resolved due to other changes I made afterward, so will test without this.